# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: widget/gtk/MozContainerWayland.h
# Commit: c766791726f1
# Full Hash: c766791726f1bbce2bb7581c62ccc8b334e8c9e2
# Author: stransky <stransky@redhat.com>
# Date: 2025-02-06 04:08:03
# Description:
#   Bug 1943662 [Wayland] Don't call GdkCeiledScaleFactor() from compositor/render threads r=emilio
#   
#   - Mark SetEGLNativeWindowSize() calls Wayland only as we need to set explicitly wl_egl_window size on Wayland.
#   - Remove moz_container_wayland_egl_window_set_size() and use WaylandSurface directly.
#   - Remove plain call of FractionalScaleFactor() from nsWindow::SetEGLNativeWindowSize() as it used for logging only and move it to logging block.
# ==============================================================================

diff -r 12d7b2544b10 -r c766791726f1 widget/gtk/MozContainerWayland.h
--- a/widget/gtk/MozContainerWayland.h	Wed Feb 05 16:34:23 2025 +0000
+++ b/widget/gtk/MozContainerWayland.h	Wed Feb 05 16:38:14 2025 +0000
@@ -58,8 +58,6 @@
     MozContainer* container);
 
 gboolean moz_container_wayland_has_egl_window(MozContainer* container);
-bool moz_container_wayland_egl_window_set_size(MozContainer* container,
-                                               nsIntSize aScaledSize);
 void moz_container_wayland_add_or_fire_initial_draw_callback(
     MozContainer* container, const std::function<void(void)>& initial_draw_cb);
 